web/portal/client/src/entities/ExternalCallFilter/Field/OutOfScheduleEnabled.tsx

```diff
@@ -0,0 +1,41 @@
+import {
+    PropertyCustomFunctionComponent,
+    PropertyCustomFunctionComponentProps,
+} from '@irontec/ivoz-ui/services/form/Field/CustomComponentWrapper';
+import {useStoreState} from "../../../store";
+
+type OutOfScheduleEnabledProps = PropertyCustomFunctionComponent<
+    PropertyCustomFunctionComponentProps<RouteTypeValues>
```

Comment:
This RouteTypeValues seems from a Copy&Pasterino

Reply:
Of course.